source/adios2/helper/adiosSystem.cpp

```diff
@@ -48,10 +48,20 @@ bool IsLittleEndian() noexcept
 
 std::string LocalTimeDate() noexcept
 {
+    struct tm now_tm;
+    char buf[30];
```

Comment:
I'd rather see this reworked to entirely use `std::chrono` for formatting rather than dropping it to the c functions.

Reply:
That works too.  Just observing that this is an independent fix that can go in at any time...

Reply:
I used the C API because I couldn't find the formatting function in C++, so I didn't think there was one, but it looks like it was on a different page from where I was looking. Poor documentation... I will fix it.